### PR TITLE
[np-47582] Add initial CloudFormation templates

### PR DESCRIPTION
**Pull request by LarsV123**
Ticket: https://sikt.atlassian.net/browse/NP-47971

Changes:

- Adds a CFN template for the root pipeline that deploys other resources. This sets up one shared pipeline that deploys a vault for each environment.
- Adds a CFN template for a secure backup vault shared with specified accounts

### CODE REVIEW

File: check.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Format the files
+echo "Formatting..."
+rain fmt ./templates/pipeline.yml -w
+rain fmt ./templates/template.yml -w
```
Comment (brinxmat) [inline check.sh:6]: Har ikke vært borti rain fmt før!

**Review by brinxmat — APPROVED** [review]
The best review of this is to deploy it.

File: check.sh (hunk 1 above)
Comment (LarsV123) [reply check.sh:6]: Fra dokumentasjonen til `rain`: 
> Rain is what happens when you have a lot of CloudFormation

Det gjør masse annet også som sikkert er relevant, men bruker bare formattering nå.

**Comment by LarsV123** [comment]
> The best review of this is to deploy it.

It's deployed to the tooling account temporarily, so the result can be inspected there.

**Pull request merged**